koschei/backend/services/resolver.py

```diff
@@ -464,12 +464,12 @@ def generate_repo(self, sack, collection, repo_id):
             .filter_by(collection_id=collection.id)\
             .delete()
         prev_state = collection.state_string
+        collection.latest_repo_id = repo_id
```

Comment:
Resolver looks at the latest_repo_id to see whether it has processed that repo already. If resolver is shut down or OOM killed or otherwise interrupted, it would never reattempt to resolve that repo.

last_repo_resolved currently has the meaning you described, but unfortunately latest_repo_id doesn't. I'm trying to improve the situation in feature/independent-repo-resolution branch. So maybe just ignore my comment for now